fluidefi/fluidefi-caspernet-aggregator-tools
# Lp Token Total Supply Service

**Summary of change:**

- Created erc20 transfer fetcher service.
- Created erc20 transfer filter service.
- Created token total supply fetcher service.
- Created token total supply record finder.
- Created token total supply calculator.
- Created token total supply saver.
- Created lp token total supply consumer.


**Steps for testing**
- Make sure your local git is up to date
```bash
git pull
```

- Checkout the branch `feature/lp-token-total-supply`
```bash
git checkout feature/lp-token-total-supply
```

- Run the test across the test-net
```bash
npm run test
```

- Confirm tests have passed.
- Run example.
